test/specs/instance.spec.ts

```diff
@@ -1936,9 +1936,9 @@ describe('instance', () => {
       ParanoidUser.create({username: 'cuss'})
         .then(() =>
           ParanoidUser.findAll<ParanoidUser>({
-            where: sequelize.and({
-              username: 'cuss'
-            })
+            where: {
+              [sequelize.Op ? sequelize.Op.and : '$and']: { username: 'cuss' }
```

Comment:
Awkward workaround to simultaneously support Sequelize v3 and v4.  

`@types/sequelize` for v4 allows _some_ of the legacy operators, but not all of them (there might be a reason for it, but I can't find one).  I'll send them a separate PR to fix that.

Reply:
In #157 the project decided to drop v3 support - however since I don't think that has happened yet, I'm not sure what the best course of action for this code is. For now what you have is probably good, since the codebase currently tries to support both - but when we drop v3 we can get rid of the awkward workaround.

Reply:
Yep, but as long as we've got tests targeting v3....